core/src/main/java/io/github/tcdl/msb/support/Utils.java

```diff
@@ -56,8 +58,8 @@ public static String toJson(Object object, ObjectMapper objectMapper) {
         try {
             return objectMapper.writeValueAsString(object);
         } catch (JsonProcessingException e) {
-            LOG.error("Failed parse to JSON from: " + object, e);
-            throw new JsonConversionException(e.getMessage());
+            LOG.error("Failed to parse to JSON object: [{}] ", object);
```

Comment:
I don't see the reason for that. 
In case i want to log messageId we need -> 
1.pass messageId to ndc + pop messageId in Utils. 
2.still log this in Producer/Consumer/DefaultChannelMonitorAggregator as we need context where this happened.

Is that what we want? or do i miss smth?

p.s. If vice versa pass context from Utils to Producer/Consumer/DefaultChannelMonitorAggregator we need to be sure after we push in Utils other component will have to pop. If we add new that use Utils that can brake and get even worth.